### PR TITLE
Feature/search data integration

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- Successfully implement the Search Bar validation 
- Currently only able to successfully search by name
- Tried ID, not able to get going - we can leave this for a future iteration

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -22,12 +23,35 @@ class App extends Component {
     })
   }
 
+  addPokemon = (queriedPokemon) => {
```
Comment (hayleyw7) [inline src/components/App/App.js:26]: I love the way this turned out.

File: src/components/Search/Search.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+import './Search.css'
 
 class Search  extends Component  {
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
         this.state = {
-            foundPokemon:''
+            queriedPokemon: ''
```
Comment (hayleyw7) [inline src/components/Search/Search.js:8]: Way better name

**Review by hayleyw7 — APPROVED** [review]

File: src/components/Search/Search.js [hunk 3]
```diff
@@ -16,25 +15,28 @@ class Search  extends Component  {
...
-                <input 
-                    type="search" 
-                    name="foundPokemon" 
-                    className="search-bar" 
-                    placeholder="Choose your Pokemon" 
+                <input
+                    type="search"
+                    name="queriedPokemon"
+                    className="search-bar"
+                    placeholder="Choose your Pokemon"
```
Comment (joanafbrito) [inline src/components/Search/Search.js:36]: I really like this indentation, looks away better tsk for paying attention to that

File: src/components/App/App.js [hunk 4]
```diff
@@ -22,12 +23,35 @@ class App extends Component {
...
+      }
+    })
+
+    if (verifiedName === undefined) {
+      console.log('No Good NAMe!')
+    } else {
+      console.log(verifiedName, 'IT WORKSSSSSSS!!!!')
+      return verifiedName
+    }
+  }
```
Comment (joanafbrito) [inline src/components/App/App.js:47]: Great thought about those functions, and I just love the console.logs 🙃

File: src/components/App/App.js [hunk 5]
```diff
@@ -9,6 +9,7 @@ class App extends Component {
     super();
     this.state = {
       pokeDex: [],
+      foundPokemon: {}
       // what will we need for favoriting?
```
Comment (joanafbrito) [inline src/components/App/App.js:13]: I really appreciate the comment here, so we know where our thoughts were

**Review by joanafbrito — APPROVED** [review]
Thank you for the all the work you have put into that 
I've comment in the code lines .

**Pull request merged**